proposals/0007-rn-versions-lifecycle-support.md

```diff
@@ -0,0 +1,69 @@
+# RFC: React Native Versions Lifecycle & Support
+
+
+
+>As we are finding ways to better support React Native community, we are moving towards supporting a limited amount of versions of React Native better.
+
+
+The React Native Core team is working to strike a balance between the speed of innovation with the amount of regular change to the platform. We understand that each release brings new value and has an adoption cost to the community.
+This document describes how we intend to formally balance these concerns and provide clear and consistent support over time.
+React Native team will work on best-effort bases to release a new version of React Native every four months. And will focus on speeding up the release cycle and put considerable effort into ensuring a seamless upgrade path.
```

Comment:
```suggestion
React Native team will work on best-effort basis to release a new version of React Native every four months. And will focus on speeding up the release cycle and put considerable effort into ensuring a seamless upgrade path.
```

Maybe instead of `React Native every four months` we could change the wording to `at least every four months`? I believe we should aim for four months to be a max, but I can also see us making release more often. That's not that clear imho from the current statement.

Reply:
Agree on this. @Andjeliko can we relax the wording?

Reply:
Included in https://github.com/react-native-community/discussions-and-proposals/pull/513